### PR TITLE
Feature/recipe view

### LINKED ISSUE #19: Build out Individual Recipe Page View

### PR DESCRIPTION
**Pull request by oacortez**
What (if any) features are you implementing?
- Added event listeners 
- display recipe  name and price
- show and hide main browser to card info
- build recipe view and style for it

What (if anything) did you refactor?
- Refactor naming for html class/ids
- Refactor naming css properties

Were there any issues that arose?
- none

Is there anything that you need from your teammate?
- n/a

Any other comments, questions, or concerns?
- n/a

### CODE REVIEW

File: src/classes/RecipeRepository.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
+import Recipe from './Recipe';
+
 class RecipeRepository {
-  constructor(recipes = []) {
-    this.recipes = recipes;
+  constructor(recipes) {
```
Comment (mr-ryan12) [inline src/classes/RecipeRepository.js:4]: Nicely done merging all the data into the class files. This cleans up our scripts file and keeps all the data in the RecipeRepository class.

File: src/scripts.js [hunk 2]
```diff
@@ -6,39 +6,57 @@ import RecipeRepository from './classes/RecipeRepository';
...
-    ));
-    return acc;
-  }, []);
-  return recipeList;
-}
+const recipeRepo = new RecipeRepository(recipeData);
+const recipeContainer = document.getElementById('browse-page');
+const recipeTitle = document.querySelector('#recipe-title');
+const price = document.querySelector('#price');
+const homePage = document.querySelector('.home-page');
```
Comment (mr-ryan12) [inline src/scripts.js:13]: Possibly an opportunity to refactor. Just need to keep a convention for consistency when targeting DOM elements. Let's decide to either use querySelector or getElementById.

File: src/classes/RecipeRepository.js [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+      acc.push(new Recipe(
+        recipe.id,
+        recipe.image,
+        recipe.ingredients,
+        recipe.instructions,
+        recipe.name,
+        recipe.tags
+      ));
+      return acc;
+    }, []);
```
Comment (philalewis) [inline src/classes/RecipeRepository.js:15]: Love that we moved the recipe class instantiation inside the RecipeRepository class file. Good seperation from DOM related elements.

File: src/index.html [hunk 4]
```diff
@@ -38,18 +38,35 @@ <h1>Whats Cookin</h1>
...
+          <h2 id="recipe-title">Title</h2>
+          <h4 id="price">Price</h4>
+          <section class="ingredients-view">
+            <h5>Ingredients</h5>
+            <ul class="ingredients-list"></ul>
+          </section>
+          <section class="directions-view">
+            <h5>Directions</h5>
+            <ol class="directions-list"></ol>
+          </section>
```
Comment (philalewis) [inline src/index.html:64]: We'll put some lists in these sections in the JS

File: src/styles.css [hunk 5]
```diff
@@ -150,4 +150,60 @@ h1 {
...
+.ingredients-view,
+.directions-view {
+  display: flex;
+  width: 30vw;
+  background-color: lightgrey;
+  margin: 1vh;
+  border-radius: 10px;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (philalewis) [inline src/styles.css:203]: I'm liking the way this section looks on the page

**Review by philalewis — APPROVED** [review]
Looking much better and we got some good functionality worked out.

**Pull request merged**